AmericanMedicalAssociation/ama-style-guide-2
# ADF-136 : Prototype doctor edit header - no image

**Jira Ticket**
- [ADF-135: Prototype doctor edit header - no image](https://palantir.atlassian.net/browse/ADF-136)

## Description

- Add Dr Finder profile header with no image.


## To Test

- Checkout the **ADF-136-dcotor-edit-header-no-image** branch and cd into styleguide, run gulp serve to spin up the pattern lab style guide.

- Verify that dr-finder-doctor-edit-header folder and related files exist.

- Visit http://localhost:3000/?p=molecules-dr-finder-doctor-edit-header-no-imager and verify modal.

## Relevant Screenshots/GIFs
<img width="647" alt="Screen Shot 2022-10-05 at 10 55 04 AM" src="https://user-images.githubusercontent.com/54925022/194117796-bf2f6574-f70a-48d0-a1c2-2e22179c4237.png">

## Related Task

- Add profileHeader-image and filterList-hover color variables in the _colors.scss file.
- Modified the dr finder filter list scss file with the new color variable.
